handlers/dimensions.go

```diff
@@ -143,6 +144,10 @@ func (f *Filter) listSelector(w http.ResponseWriter, req *http.Request, name str
 type Range struct {
 	Start          string `schema:"start"`
 	End            string `schema:"end"`
+	StartMonth     string `schema:"start-month"`
```

Comment:
For ease of finding your variables in the future, keep variables in an alphabetical order unless there is a better logical layout.